mtav7.0.2/application.properties

```diff
@@ -0,0 +1,6 @@
+quarkus.kogito.devservices.enabled=false
+
+mta.url = ${MTA_URL:http://mta-ui.openshift-mta.svc.cluster.local:8080}
+quarkus.rest-client.mta_json.url = ${mta.url}
```

Comment:
if you look at the original file in mta folder, it is:
`${mta.url}/hub`

Reply:
Please check: https://docs.google.com/document/d/1Zmv8mGSKA_fQzC3J38d5jYaJ28EmGTIRE8jIPmLBCv4/edit?usp=sharing

Reply:
Then I give up, surely @gabriel-farache knows it